code/parachain/frame/staking-rewards/src/test/test_reward_accumulation_hook.rs

```diff
@@ -148,6 +139,7 @@ fn test_accumulate_rewards_pool_empty_refill() {
 		// cap will be hit after 5_000 seconds (834 blocks)
 		const A_A_REWARD_RATE: u128 = A::units(2) / 100_000;
 		const A_A_MAX_REWARDS: u128 = A::units(1) / 10;
```

Comment:
as far as I can tell, this is unused now? you should also remove the comments about the cap being hit

Reply:
You are right. I rephrased those comments as `initial amount will be fully rewarded after 25_000 seconds (4167 blocks)`, so that there's some explanation within the code about why we later use `progress_to_block(STARTING_BLOCK + 4167, &mut current_block);`

Reply:
Sounds good :+1: